### PR TITLE
[WINDUP-3246]: WINDUP distribution of CLI with Konveyor themed Reports

### PR DESCRIPTION
**Pull request by carlosthe19916**
https://issues.redhat.com/browse/WINDUP-3246

### CODE REVIEW

File: .github/workflows/community_distribution_build.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          java-package: jdk
+          cache: 'maven'
+
+      - name: Build Windup Rulesets
+        run: mvn clean install -DskipTests -Dcommunity -f windup-rulesets/
+      - name: Build Windup
```
[the remark this answers is not in the document]
Comment (carlosthe19916) [reply .github/workflows/community_distribution_build.yml:33]: done

**Review by jmle — APPROVED** [review]

**Comment by jmle** [comment]
Hi @carlosthe19916, the PR looks good, although I would refrain from duplicating the script files - maybe we can create two different script files that just call the first one with different parameters for each build?

**Comment by carlosthe19916** [comment]
> Hi @carlosthe19916, the PR looks good, although I would refrain from duplicating the script files - maybe we can create two different script files that just call the first one with different parameters for each build?

@jmle good suggestion about not duplicating script content. Give me some time and I will fix it, please.

**Comment by carlosthe19916** [comment]
@jmle Thanks for the previous suggestion. My last commit changed a couple of things:
- Since the mta-cli and windup-cli have exactly the same content then there is no need to duplicate files. Each Theme will use a different assembly.xml file but the files content will remain as they are.

When you have time please review this PR again to see if there is something to change or it is ready to be merged. You can build the project using `mvn clean install` for the default MTA theme or `mvn clean install -Dcommunity` for the Windup  theme.

**Comment by jmle** [comment]
Hi @carlosthe19916, could you please resolve the conflicts? Thanks

**Pull request merged**